cmd/release/build.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func build(date, tag string) ([]string, error) {
+	outputs := []string{}
+	for goos, goarchs := range targets {
+		for _, goarch := range goarchs {
+			log.Println(fmt.Sprintf("==> Build on %s for %s ...", goos, goarch))
+			target := fmt.Sprintf("dist/terraform-provider-ansiblevault_%s-%s_%s", goos, goarch, tag)
+			_, err := executeCmd(
+				[]string{
+					fmt.Sprintf("GOARCH=%s", goarch),
+					fmt.Sprintf("GOOS=%s", goos),
+					"CGO_ENABLED=0",
+				},
+				"go",
+				"build",
+				"-ldflags",
+				fmt.Sprintf("-s -w -X %s.Version=%s -X %s.BuildDate=%s", gitRepoURL, tag, gitRepoURL, date),
+				"-o",
```

Comment:
Also add `-installsuffix nocgo`

Reply:
done 3827ae5

Reply:
Sorry, it was not clear. It should be done this way https://github.com/MeilleursAgents/serge/blob/master/Makefile#L88.

It's not a `ldflags` but a real cli flag

Reply:
🔥

Reply:
Available in 3d4fbc1